.travis.yml

```diff
@@ -14,12 +14,21 @@ script:
   - npm test
   - npm run coverage
 
-addons:
-  firefox: 'latest'
 
 before_script:
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
 
 after_success:
   - npm run coverage-publish
+
+env:
+  - CXX=g++-4.8
```

Comment:
So that travis can build native modules (node 4 and greater require a c++ compiler for those) Details here: https://docs.travis-ci.com/user/languages/javascript-with-nodejs#Node.js-v4-(or-io.js-v3)-compiler-requirements